samples/grids/grid/infinite-scroll/App.razor

```diff
@@ -0,0 +1,131 @@
+@using IgniteUI.Blazor.Controls
+@using System.Net.Http;
+@using Newtonsoft.Json;
+
+<IgbGrid AutoGenerate="false"
```

Comment:
the grid should be wrapped in DIV, e.g.
<div class="container vertical">

Reply:
@HUSSAR-mtrela  Ok. But what is the reason it needs to be wrapped in a div? I haven't noticed any issues when there's no div.

Reply:
to ensure that all samples have the same styling on their container and CSS styling from docs does not show up in our samples